packages/tocco-ui/src/MultiCheckbox/MultiCheckbox.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+const checkboxState = {
+  CHECKED: 'checked',
+  INDETERMINATE: 'indeterminate',
+  UNCHECKED: 'unchecked'
+}
+
+/**
+ * Use <MultiCheckbox> to display if none, some or all items of a portion are selected.
+ */
+class MultiCheckbox extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {status: this.props.status || false}
+    this.onChange = this.onChange.bind(this)
+  }
+
+  componentDidMount() {
+    this.handleIndeterminate(this.state.status)
+  }
+
+  componentDidUpdate() {
+    this.handleIndeterminate(this.state.status)
+  }
+
+  handleIndeterminate(status) {
+    // eslint-disable-next-line react/no-find-dom-node
+    ReactDOM.findDOMNode(this).indeterminate = status === checkboxState.INDETERMINATE
```

Comment:
As i have written in another PR: reconsider using eslint disable because usually there is something wrong with the code if you need to use eslint
in this case you should have used reacts ref to get the dom element.